python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -151,6 +152,7 @@ def repackage(self, phosim_amp_files, out_dir='.'):
         parts = chip_id.split('_')
         raft = parts[0]
         ccd = parts[1]
+        snap = int(os.path.basename(fn).split('_')[7][1:4])
```

Comment:
On balance, I think I'd prefer the argument here to be `phosim_amp_files[0]` rather than `fn`. Firstly because in the comment at the start of this section you explicitly say first file, and secondly I always feel a bit wary of relying on the value of a loop variable.